frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/copy_and_paste/custom_paste_command.dart

```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: use_build_context_synchronously
```

Comment:
Use context.mount before using context async, instead of ignoring it.

Reply:
It still complains because I moved it to a function since it is reused a lot. So it is guarded.